CSharp/TextSearch/readme.md

```diff
@@ -1,22 +1,22 @@
 ## TextSearch
 
 ### About this sample
-This map-reduce style sample uses Azure Batch to perform parallel text processing on an input file by splitting it up into multiple sub-files and performing regular expression matching on each sub-file. The results are then rolled-up into a final report by a reduction phase where it is uploaded to Azure Storage.
+This map-reduce style sample uses Azure Batch to perform parallel text processing on an input file by splitting it up into multiple sub-files and performing regular expression matching on each sub-file.
+This sample makes use of a variety of Azure Batch features, such as auto-pool functionality, task `ResourceFiles` for moving data into VMs, task `OutputFiles` for moving data out of VMs, and task dependencies for specifying tasks relationships to one another.
```

Comment:
Two new lines between new lines?

Reply:
Got rid of the line break actually as it's not really needed. The final text reads fine as one paragraph I think.